### DISCUSSION TITLE
Problem running 3D Onera wing case

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#276] [Tutorials]
[logOpt.txt](https://github.com/mdolab/dafoam/files/8126336/logOpt.txt)
Hi
My computer has  16Gb RAM and 8 CPU cores and I can not run 3D wing optimization of my own wing. I run the Onera Wing in tutorials successfully but when I run my own case customized within Onera example, It stops with this error:

mpirun noticed that process rank 2 with PID 1063 on node 3931db16682e exited on signal 9 (Killed).

If it is memory problem, why the basic Onera example runs successfully?
My mesh file is also not very fine, and it is unstructured SnappyHexMesh. 
I have attached the Log file for your consideration.

I would appreciate it if you help me in this regard

[logOpt.txt](https://github.com/mdolab/dafoam/files/8126347/logOpt.txt)

**Comment by friedenhe** [comment]
Check: https://dafoam.github.io/mydoc_get_started_faq.html#how-many-cpu-cores-to-use-and-how-much-memory-does-it-need
